src/server/models/listItem/providers/CovidTestSupplier.ts

```diff
@@ -44,7 +41,7 @@ export async function findPublishedCovidTestSupplierPerCountry(props: {
       offset,
     });
 
-    return await prisma.$queryRaw(query);
+    return await prisma.$queryRawUnsafe(query);
```

Comment:
same question: why use queryRawUnsafe here if it was working before.  Potential double-escaping?

Reply:
This is a fair question. In Prisma version 2x this was working fine but after upgrading to 3x this no longer worked. I don't have any screenshots of the error but I'm happy to present some on request.